### DISCUSSION TITLE
Dodge Mod Compatibility Patches

### DISCUSSION THREAD

**Discussion by Monitor221hz** [Monitor221hz/Pandora-Behaviour-Engine-Plus#292] [Patches]
[TK Dodge](https://www.nexusmods.com/skyrimspecialedition/mods/111788)

[The Ultimate Dodge Mod](https://www.nexusmods.com/skyrimspecialedition/mods/127728?tab=description)

**Comment by BryanAldrich** [comment]
Is there a patch for https://www.distaranimation.com/mods/dodge?
